app/layout.tsx

```diff
@@ -34,15 +36,15 @@ export default function RootLayout({
   const contentCss = 'content bg-slate-100 px-10 lg:px-0 grow overflow-y-auto';
 
   return (
-    <html>
+    <html suppressHydrationWarning>
```

Comment:
I created a task to add a description comment to each suppressHydrationWarnings. So we explain to other devs why we used it. 
https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/issues/34